### DISCUSSION TITLE
no script wants to work

### DISCUSSION THREAD

**Discussion by AlexandreCA** [tteck/Proxmox#1732] [General]
Hello. 

I am not connected in root but I should have the same power as a root in my Proxmox session. The reason is that no script wants to work

for example:    ________                ________                    __
  / ____/ /___  __  ______/ / __/ /___ _________  ____/ /
 / /   / / __ \/ / / / __  / /_/ / __ `/ ___/ _ \/ __  / 
/ /___/ / /_/ / /_/ / /_/ / __/ / /_/ / /  /  __/ /_/ /  
\____/_/\____/\__,_/\__,_/_/ /_/\__,_/_/   \___/\__,_/   
                                                         
Loading...
ipcc_send_rec[1] failed: Unknown error -1
ipcc_send_rec[2] failed: Unknown error -1
ipcc_send_rec[3] failed: Unknown error -1
Unable to load access control list: Unknown error -1

[ERROR] in line 428: exit code 255: while executing command NEXTID=$(pvesh get /cluster/nextid)

alexandre@pve:~$ 

and that, doesnt matter the script.

**Comment by tteck** [comment]
Are you executing the script directly within the Proxmox shell?

**Reply by AlexandreCA** [reply]
Yes, but by chance, I used the sudo command before and strangely script works.

**Reply by tteck** [reply]
The default user for Proxmox is root. Have you introduced an additional user to Proxmox?

**Reply by AlexandreCA** [reply]
yes

**Reply by tteck** [reply]
Could you please explain the reason for this?

**Reply by AlexandreCA** [reply]
Honestly, I'm new to proxmox, and I read a tutorial that said it was not recommended to be rooted.

**Reply by tteck** [reply]
Simply switch the user back to root before executing any scripts from this repository.

**Reply by AlexandreCA** [reply]
That's a fact! But it's a false alarm, everything works :) Your scripts are great though.
